index.html

```diff
@@ -0,0 +1,232 @@
+            characters in identifiers. Some examples of legal names are
+            Number_hits, temp99, and _name.
+          </p>
+        </article>
+      </section>
+      <section class="main-section" id="Declaring_Variables">
+        <header><h2>Declaring Variables</h2></header>
+        <article>
+          You can declare a variable in three ways:
+          <p>
+            With the keyword var. For example,<br />
+            <code>var x = 42.</code><br />
+            This syntax can be used to declare both local and global variables.
+          </p>
+          <p>
+            By simply assigning it a value. For example,<br />
+            <code>x = 42.</code><br />
+            This always declares a global variable. It generates a strict
+            JavaScript warning. You shouldn't use this variant.
+          </p>
+          <p>
+            With the keyword let. For example,<br /><code> let y = 13.</code
+            ><br />
+            This syntax can be used to declare a block scope local variable. See
+            Variable scope below.
+          </p>
+        </article>
+      </section>
+      <section class="main-section" id="while_statement">
+        <header>while statement</header>
+        <article>
+          A while statement executes its statements as long as a specified
+          condition evaluates to true. A while statement looks as follows:<br />
+
+          <code>while (condition) statement</code> <br />If the condition
+          becomes false, statement within the loop stops executing and control
+          passes to the statement following the loop.
+
+          <p>
+            The condition test occurs before statement in the loop is executed.
+            If the condition returns true, statement is executed and the
+            condition is tested again. If the condition returns false, execution
+            stops and control is passed to the statement following while.
+          </p>
+
+          <p>
+            To execute multiple statements, use a block statement ({ ... }) to
+            group those statements.
+          </p>
+
+          Example:
+
+          <p>
+            The following while loop iterates as long as n is less than three:
+          </p>
+
+          <code>var n = 0; var x = 0; while (n &lt; 3) { n++; x += n; }</code>
+          <p>
+            With each iteration, the loop increments n and adds that value to x.
+            Therefore, x and n take on the following values:
+          </p>
+
+          <ul>
+            <li>After the first pass: n = 1 and x = 1</li>
+            <li>After the second pass: n = 2 and x = 3</li>
+            <li>After the third pass: n = 3 and x = 6</li>
+          </ul>
+          <p>
+            After completing the third pass, the condition n &lt; 3 is no longer
+            true, so the loop terminates.
+          </p>
+        </article>
+      </section>
+    </main>
```

Comment:
You did great work on how you used sections and article tags however you may consider using one section tag and the other sections as articles.

Reply:
[ ] Kindly phrase your comments politely, as suggestions instead of orders. example:`"I think it would be better to use semantic tags"`
[ ] Kindly include an explanation, a "why", for each requested change. For example:`Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible

Reply:
- [ ] As suggested by the preview reviewer. Kindly make sure to phrase your `comment` politely and provide some context on why the change you're requesting is necessary. This can help the student 👨‍🎓 understand the reasoning behind the change and prevent them from making similar mistakes in the future. By explaining the importance of the change, the student 👨‍🎓 can also learn more effectively and improve their coding skills. Keep up the good work!